File: 3055-maximum-odd-binary-number/3055-maximum-odd-binary-number.py
class Solution(object):
    def maximumOddBinaryNumber(self, s):
        """
        :type s: str
        :rtype: str
        """
        one=s.count('1')-1
        zero=s.count('0')
        t='1'*(one) + '0'*zero
        t=t+'1'
        print(t)
        return t
        

File: 3055-maximum-odd-binary-number/3055-maximum-odd-binary-number.java
class Solution {
    public String maximumOddBinaryNumber(String s) {
        int no_ones=s.length()-s.replace("1","").length()-1;
        int no_of_zeros=s.length()-s.replace("0","").length();
        StringBuilder t = new StringBuilder();
        for(int i=0;i<no_ones;i++){
                t.append("1");
        }
        for(int i=0;i<no_of_zeros;i++){
                t.append("0");
        }
        t.append("1");
        return t.toString();
    }
}